cps/main.c

```diff
@@ -1229,10 +1232,22 @@ run_cps(struct net_config *net_conf, struct gk_config *gk_conf,
 		goto stage3;
 	}
 
+	cps_conf->arp_mp = (struct rte_mempool *)rte_mempool_create(
```

Comment:
Why is this cast needed since the return of `rte_mempool_create()` already is `struct rte_mempool *`?

This comment also applies to the ND patch.

Reply:
You are right.